pkg/lite-apiserver/proxy/controller.go

```diff
@@ -169,18 +168,10 @@ func (c *RequestCacheController) doRequest(r *http.Request) {
 	newReq.Header.Set(EdgeUpdateHeader, time.Now().String())
 	defer newReq.Body.Close()
 
-	resp, err := client.Do(newReq)
+	_, err = client.Do(newReq)
 	if err != nil {
 		klog.Errorf("auto update request do err %v", err)
 	}
-	defer func() {
-		if resp != nil {
-			resp.Body.Close()
-		}
-	}()
-	if _, err := io.Copy(ioutil.Discard, resp.Body); err != nil {
-		klog.Errorf("io copy err: %s", err.Error())
-	}
```

Comment:
it may cause network connect leakage if deleting this.

Reply:
> it may cause network connect leakage if deleting this.

I've changed it back, please take a look again.